drkafka/src/main/java/com/pinterest/doctorkafka/config/DoctorKafkaConfig.java

```diff
@@ -35,6 +35,7 @@
       "action.broker_replacement.interval.seconds";
   private static final String BROKER_REPLACEMENT_COMMAND = "action.broker_replacement.command";
   private static final String OSTRICH_PORT = "ostrich.port";
+  private static final String RESTART_DISABLE = "restart.disabled";
```

Comment:
Could you update https://github.com/pinterest/doctorkafka/blob/master/drkafka/config/doctorkafka.properties#L73  with info on this option?